code/game/machinery/cryopod.dm

```diff
@@ -0,0 +1,464 @@
+		return
+
+	if(occupant)
+		to_chat(user, "<span class='boldnotice'>The cryo pod is already occupied!</span>")
+		return
+
+	if(target.stat == DEAD)
+		to_chat(user, "<span class='notice'>Dead people can not be put into cryo.</span>")
+		return
+
+	if(target.client && user != target)
+		to_chat(user, "<span class='notice'>You can't put conscious people into [src]!</span>")
+	else if(target.client)
+		if(alert(target,"Would you like to enter cryosleep?",,"Yes","No") == "No")
+			return
+
+	var/generic_plsnoleave_message = " Please adminhelp before leaving the round, unless there are no administrators online."
+
+	if(target == user && world.time - target.client.cryo_warned > 5 * 600)//if we haven't warned them in the last 5 minutes
+		if(target.mind.assigned_role in GLOB.command_positions)
+			to_chat(target, "<span class='danger'>You're a Head of Staff![generic_plsnoleave_message]</span>")
+		if(iscultist(target) || is_servant_of_ratvar(target))
+			to_chat(target, "<span class='danger'>You're a Cultist![generic_plsnoleave_message]</span>")
+		if(istype(SSticker.mode, /datum/game_mode/blob))
+			var/datum/game_mode/blob/G = SSticker.mode
+			if(target.mind in G.blob_overminds)
+				to_chat(target, "<span class='danger'>You're a Blob![generic_plsnoleave_message]</span>")
+		if(is_devil(target))
+			to_chat(target, "<span class='danger'>You're a Devil![generic_plsnoleave_message]</span>")
+		if(is_gangster(target))
+			to_chat(target, "<span class='danger'>You're a Gangster![generic_plsnoleave_message]</span>")
+		if(istype(SSticker.mode, /datum/game_mode/revolution))
+			var/datum/game_mode/revolution/G = SSticker.mode
+			if(target.mind in G.head_revolutionaries)
+				to_chat(target, "<span class='danger'>You're a Head Revolutionary![generic_plsnoleave_message]</span>")
+			else if(target.mind in G.revolutionaries)
+				to_chat(target, "<span class='danger'>You're a Revolutionary![generic_plsnoleave_message]</span>")
+		target.client.cryo_warned = world.time
+		return
+
+	if(target == user)
+		visible_message("[user] starts climbing into the cryo pod.")
+	else
+		visible_message("[user] starts putting [target] into the cryo pod.")
+
+	if(occupant)
+		to_chat(user, "<span class='boldnotice'>\The [src] is in use.</span>")
+		return
+	close_machine(target)
+
+	to_chat(target, "<span class='boldnotice'>If you ghost, log out or close your client now, your character will shortly be permanently removed from the round.</span>")
+	name = "[name] ([occupant.name])"
+	log_admin("<span class='notice'>[key_name(target)] entered a stasis pod.</span>")
+	message_admins("[key_name_admin(target)] entered a stasis pod. (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
+	add_fingerprint(target)
+
+//Attacks/effects.
+/obj/machinery/cryopod/blob_act()
+	return //Sorta gamey, but we don't really want these to be destroyed.
+
+// /obj/machinery/computer/cryopod/robot
```

Comment:
Kinda need it for cyborgs.

Reply:
The point is to let Cyborgs use regular cryo instead of the AI sat. It's more easily accessible.

I can still add the robot storage if you want me to, but I think it's a wasted effort...

Reply:
👌

Reply:
Have you tested the regular ones working properly, and it not storing anything it shouldn't?

If so - lets remove all this commented out code entirely.